### PR TITLE
GH readme update for Gloo Gateway

### PR DESCRIPTION
**Pull request by Nadine2016**
# Description

Please include a high level summary of the changes.

This bug fixes ... \ This new feature can be used to ...

_Fill out any of the following sections that are relevant and remove the others_
## API changes
- Added x field to y resource
- ...

## Code changes
- Fix error in `Foo()` function
- Add `Bar()` function
- ...

## CI changes
- Adjusted schedule for x job
- ...

## Docs changes
- Added guide about feature x to public docs
- Updated README to account for y behavior
- ...

# Context

Users ran into this bug doing ... \ Users needed this feature to ...

See slack conversation [here](https://solo-io-corp.slack.com/archives/some/post)

## Interesting decisions
 
We chose to do things this way because ...

## Testing steps

I manually verified behavior by ...

## Notes for reviewers

Be sure to verify intended behavior by ...

Please proofread comments on ...

This is a complex PR and may require a huddle to discuss ...

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: devel/README.md [hunk 1]
```diff
@@ -9,6 +9,6 @@ Find tools and information to help you develop the Gloo Gateway project.
...
 
 Other resources:
-* [Developer guide](https://docs.solo.io/gloo-edge/latest/guides/dev/) to set up your development environment and learn more about extending the functionality of the Gloo Gateway project and related plug-ins. While written for external contributors, internal Solo developers might also find this guide helpful when onboarding.
-* [Product documentation](https://docs.solo.io/gloo-edge/latest/) with guides for end users to use Gloo Gateway as a product
-* [Guide to contribute to the documentation](https://docs.solo.io/gloo-edge/latest/contributing/documentation/)
\ No newline at end of file
+* [Developer guide (Gloo Edge API)](https://docs.solo.io/gloo-edge/latest/guides/dev/) to set up your development environment and learn more about extending the functionality of the Gloo Gateway project and related plug-ins. While written for external contributors, internal Solo developers might also find this guide helpful when onboarding.
+* [Product documentation (K8s Gateway API)](https://docs.solo.io/gateway/latest/)
```
Comment (Rachael-Graham) [inline devel/README.md:13]: is K8s alright? If youve been using it consistently in the docs that's fine, just making sure

File: devel/contributing/documentation.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
 # Documentation
 
-The docs for Gloo Gateway are built from the contents that can be found in the [docs/content](/docs/content) directory of this repository.
+The documentation is built from a private GitHub. Currently, only Solo.io team members can contribute to this repository.
```
Comment (Rachael-Graham) [inline devel/contributing/documentation.md:3]:
```suggestion
The documentation is built from a private GitHub repository. Currently, only Solo.io team members can contribute to this repository. 
```

File: devel/contributing/documentation.md [hunk 3]
```diff
@@ -1,12 +1,10 @@
 # Documentation
 
-The docs for Gloo Gateway are built from the contents that can be found in the [docs/content](/docs/content) directory of this repository.
+The documentation is built from a private GitHub. Currently, only Solo.io team members can contribute to this repository. 
 
-Improving the documentation, including by adding examples or use cases, is the easiest way to contribute to Gloo Gateway. If you see a piece of content that can be better, open a PR with an improvement, it doesn't matter how small!
-
-For more information on how to contribute to the documentation, check out the contribution guidelines in the [Gloo Gateway documentation](https://docs.solo.io/gloo-edge/latest/contributing).
+To request changes to the documentation or report an issue, follow the steps in [filing issues](/devel/contributing/issues.md)
```
Comment (Rachael-Graham) [inline devel/contributing/documentation.md:5]: maybe this?
```suggestion
To request changes to the documentation or report a problem, follow the steps to [file an issue](/devel/contributing/issues.md)
```

**Review by Rachael-Graham — APPROVED** [review]

File: devel/README.md (hunk 1 above)
Comment (Nadine2016) [reply devel/README.md:13]: yes. keith asked me to use that on the docs landing page too

**Review by artberger — APPROVED** [review]

**Review by Rachael-Graham — APPROVED** [review]

**Pull request merged**